conftest.py

```diff
@@ -264,6 +303,68 @@ def func(arglist, *, expect=0):  # , suffix=None):
     return func
 
 
+@pytest.fixture()  # Must be function scope since uses monkeypatch
+def run_cmdline_textconv(monkeypatch):
```

Comment:
Instead of duplicating this fixture with a different hard-coded command, I would rather generalize the existing `run_cmdline_test()` by having it not assume what the command is, and have all of the points-of-use of the fixture just pass in the command along with the other arguments.

(It's possible that this isn't feasible, I haven't looked closely. Interested in your thoughts.)